src/sentry/incidents/utils/metric_issue_poc.py

```diff
@@ -76,6 +76,7 @@ def create_or_update_metric_issue(
         "timestamp": incident.date_started.isoformat(),
         "platform": project.platform or "",
         "received": incident.date_started.isoformat(),
+        "contexts": {"metric_alert": {"alert_rule_id": incident.alert_rule.id}},
```

Comment:
do we need this in both places?

Reply:
adding it to both for now while we test the UI changes - will keep it to just one when it's replaced by detector id